pages/style.css

```diff
@@ -0,0 +1,169 @@
+.index-div-one {
+    width: 100vw;
+    height: 100vh;
+    background-image: url(images/portfolio_index_top_image.jpg);
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position:center;
+}
+
+.index-div-two {
+    -webkit-clip-path: polygon(100vw 0, 0% 100%, 100vw 100vh);
+    clip-path: polygon(100vw 0, 0% 100vh, 100vw 100vh);
+    background-image: url(images/portfolio_index_bottom_image.jpg);
+    width: 100vw;
+    height: 100vh;
+    background-repeat:no-repeat;
+    background-size:cover;
+    background-position:bottom;
+}
+
+div {
+    position: absolute;
+}
+
+#index-box {
+    position: absolute; 
+    top: 50%; 
+    left: 50%; 
+    transform:translate(-50%, -50%); 
```

Comment:
In general, see if you can position things with methods other than translate. Translate is good for animations, but can sometimes result in hard to understand behavior.